<img src="https://coursesandconferences.wellcomeconnectingscience.org/wp-content/themes/wcc_courses_and_conferences/dist/assets/svg/logo.svg" width="300" height="50"> 

# Genomics & Epidemiological Surveillance of Bacterial Pathogens

[Wellcome Connecting Science GitHub Home Page](https://github.com/WCSCourses) <br /> 
[Wellcome Connecting Science Website](https://coursesandconferences.wellcomeconnectingscience.org/)

Learn how to apply genomics to epidemiological surveillance of bacterial pathogens and public health.

# Course background

Established in 2013, the course provides skills to combine epidemiological, metadata and genomics data analysis, hence contributing to generating the most reliable information available for public health decision making. In the last few years, the COVID-19 pandemic challenge accelerated the introduction of advances in genomics, sequencing technology and our ability to interpret high resolution sequence data for surveillance purposes. We are now seeing a shift from the use of molecular typing data for epidemiological surveillance and public health investigations of infectious disease to using Whole Genome Sequencing (WGS) for this purpose. In some settings WGS is now being used to define outbreaks and trace infection sources in humans, animals and the environment, contributing to reinforce a One Health approach. This approach also offers enormous promise for predicting antimicrobial resistance phenotypes.

Existing studies that have used genomics for mapping the spread of pathogens have highlighted two things: Firstly, genomic datasets are growing day by day and they offer the context against which to understand local patterns of disease. Secondly, these studies highlight the power gained from public health scientists and academic research scientists working together to better understand infectious diseases. This is the foundation of this course and expert instructors have been assembled from a wide range of settings in Latin America and Europe, including national and international public health organisations and front line high-profile genomic research organisations. We recognise the synergy between academic research and public health and this is your opportunity to strengthen this effort by attending the course.

# Programme

This course builds upon previous versions, incorporating genomics and problem solving exercises with modules that demonstrate how epidemiological information and phenotyping – in combination with sequence data – can be used:

1. for georeferencing (also known as ‘phylogeography’) where the location of disease causing strains are mapped with detailed genetic and phenotypic data (for example antibiotic resistance patterns) to look for regional patterns of disease, and
2. to understand the fine detail of new and emerging infections, the risk to public health and the management of infectious diseases.

**The week-long programme will include:**

- Practical bioinformatics skills needed to manipulate raw sequence data, including: mapping and de novo assemblies, SNP calling, annotation, antimicrobial resistance calling, and analysis of genomic information
- Principles of phylogeny and construction of accurate phylogenies
- Group exercises designed to teach everyday skills required to work in genomic surveillance
- Practical exercises for using GPS to map cases of infectious disease and phenotypes (such as AMR) across the world

# Learning outcomes

After attending the course, participants should be able to:

- Select and apply appropriate tools to analyse genome sequence data effectively
- Perform comparative genomics on multiple genome sequences
- Interpret phylogenetics to gain insights into pathogen evolution and epidemiology 
- Formulate projects using the global surveillance platform Microreact and Epicollect
- Explain, using examples, how bacterial pathogens and AMR spread among human populations.

**Course Runs**      

| Course Date | Course Title | Location |Citation DOI |
|-------------|--------------|----------|-------------|
| 16–21 April 2023 | [Genomics & Epidemiological Surveillance of Bacterial Pathogens - Latin America and the Caribbean](https://coursesandconferences.wellcomeconnectingscience.org/our-events/past-events/) | Asunción, Paraguay |[![DOI](https://zenodo.org/badge/833162816.svg)](https://zenodo.org/doi/10.5281/zenodo.12806771) |
| 21–26 July 2024  | [Genomics & Epidemiological Surveillance of Bacterial Pathogens - Latin America and the Caribbean](https://coursesandconferences.wellcomeconnectingscience.org/event/genomics-epidemiological-surveillance-of-bacterial-pathogens-latin-america-and-the-caribbean-20240721/) | San José, Costa Rica| [![DOI](https://zenodo.org/badge/833162816.svg)](https://zenodo.org/doi/10.5281/zenodo.14043144)  |

******

## Citing and Re-using Course Material

The course data are free to reuse and adapt with appropriate attribution. All course data in these repositories are licensed under the <a rel="license" href="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</a>. <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons Licence" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-sa/4.0/88x31.png" /></a><br /> 

## Interested in attending a course?

Take a look what courses are coming up at [Wellcome Connecting Science Courses & Conference Website](https://coursesandconferences.wellcomeconnectingscience.org/our-events/).

---

For more information or queries, feel free to contact us via the [Wellcome Connecting Science website](https://coursesandconferences.wellcomeconnectingscience.org).

<br /> 

Find us on socials [Wellcome Connecting Science Linktr](https://linktr.ee/eventswcs)

[Wellcome Connecting Science GitHub Home Page](https://github.com/WCSCourses) 
